src/nemo-dbus/dbus.cpp

```diff
@@ -113,9 +113,6 @@ QVariant demarshallDBusArgument(const QVariant &val, int depth)
     } else if (type == qMetaTypeId<QDBusSignature>()) {
         /* Convert QDBusSignature to QString */
         res =  val.value<QDBusSignature>().signature();
-    } else if (type == qMetaTypeId<QDBusUnixFileDescriptor>()) {
-        /* Convert QDBusUnixFileDescriptor to int */
-        res =  val.value<QDBusUnixFileDescriptor>().fileDescriptor();
```

Comment:
Could you leave the branch in but not do conversion like in the `type == val.type()` case. The default statement does that too, but it prints a warning as well which I don't think you want?

Reply:
Yeah that's true. Restored the branch.